app/grandchallenge/components/tasks.py

```diff
@@ -979,3 +981,40 @@ def civ_value_to_file(*, civ_pk):
         )
         civ.value = None
         civ.save()
+
+
+def _get_sitk_image(image_file):
+    with NamedTemporaryFile() as tmp:
```

Comment:
The `zraw` file needs to have the correct name as it is referenced by the header. The filename should be preserved, you can ensure it's safe to write by using `safe_join`